gdk4-wayland/sys/src/manual.rs

```diff
@@ -0,0 +1,4 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[cfg(any(feature = "v4_4", feature = "dox"))]
```

Comment:
I don't think you need the feature="dox" here, it's what causes the docs job to fail

Reply:
I think we were just missing some doc_cfg feature on the crate. Should be fixed.